main/lsp/requests/code_action.cc

```diff
@@ -142,15 +142,27 @@ unique_ptr<ResponseMessage> CodeActionTask::runRequest(LSPTypecheckerInterface &
                     }
                     auto action = make_unique<CodeAction>("Move method to a new module");
                     action->kind = CodeActionKind::RefactorExtract;
+
                     bool canResolveLazily = config.getClientConfig().clientCodeActionResolveEditSupport &&
                                             config.getClientConfig().clientCodeActionDataSupport;
+                    auto newModuleLoc = getNewModuleLocation(gs, *def, typechecker);
+                    auto renameCommand = make_unique<Command>("Rename Symbol", "sorbet.rename");
```

Comment:
What happens if the client doesn't support this command (e.g., old VS Code extension version, or using Vim or some other language client)

Reply:
It depends on the client, VSCode will show a warning popup like this, but the edits will be applied.
<img width="572" alt="Screen Shot 2022-04-07 at 14 53 48" src="https://user-images.githubusercontent.com/9540120/162216846-41c61060-222c-48d8-893d-cdbfc3a161e4.png">

NeoVim's LSP client just ignores the command

https://user-images.githubusercontent.com/9540120/162217207-b31c4625-829c-4750-a278-4c8f58f23274.mov

Reply:
Perfect thanks